sdk/messaging/azeventhubs/README.md

```diff
@@ -23,41 +28,35 @@ Install the Azure Event Hubs client module for Go with `go get`:
 go get github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs
 ```
 
-### Prerequisites
-
-- Go, version 1.18 or higher
-- An [Azure subscription](https://azure.microsoft.com/free/)
-- An [Event Hub namespace](https://docs.microsoft.com/azure/event-hubs/).
-- An Event Hub. You can create an event hub in your Event Hubs Namespace using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create), or the [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli).
-
 ### Authenticate the client
 
 Event Hub clients are created using an Event Hub a credential from the [Azure Identity package][azure_identity_pkg], like [DefaultAzureCredential][default_azure_credential].
```

Comment:
I would like to re-write this entire H3's content (including the H4s) to fix some grammar (passive voice in the first sentence), to describe how to authenticate more detail, and to provide more context to the code examples. However, I don't have the subject matter expertise to reword this section. Please think about editing this for clarity. Thank you!